src/ui/hooks/users.ts

```diff
@@ -72,7 +72,7 @@ export function useGetUserInfo() {
   const picture: string = data?.viewer?.user.picture;
   const email: string = data?.viewer?.email;
   const internal: boolean = data?.viewer?.internal;
-  const nags: Nag[] = data?.viewer?.nags;
+  const nags: Nag[] = data?.viewer?.nags || [];
```

Comment:
This may be problematic. The nags is a list of things that tell us what not to display. An empty nags array would default to showing all the nags.

If that's the intention, then this works great. Just wanted to make sure that's the case!

Reply:
gosh - good catch.

The problem I'm trying to solve is this line
`!nags.includes(Nag.FIRST_REPLAY_2)` where we don't have an array. The better solution would be to wrap this stuff in a helper.